README.md

```diff
@@ -1,14 +1,76 @@
+![](https://img.shields.io/badge/Name-CoachMe-brightgreen?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Version-v0.0.1-blueviolet?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Stack-MERN-important?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Team-chingu.v35.bears.04-informational?style=plastic&labelColor=inactive)
 
-And before we go there's "one more thing"! Once you decide what to include
-in your `readme` feel free to replace the text we've provided here.
+<br>
 
-> Own it & Make it your Own!
+## Overview
+
+Attending Gyms are great and a good way not only to workout on our body goals but also connect with various people, but what happens when you are not so chanced?, what happens when your trainer is away for a while? what happens if it is rainy and there is no way to get to the gym to hit your body goals?. Coach Me is made with this problems in mind, trainees and trainers now have a platform where they can talk, share tips and give feedback and also trainers can manage his clients properly.
+
+This project is currently under development by a remote team in the ongoing `chingu` voyage 40.
+
+<br>
+
+## Features ✨✨✨
+The features of the `Coach Me` app are -
+- A Dashboard that connects personal trainees and their clients.
+- Personal trainers are able to
+    * Manage all clients bookings
+    * Send emails / messages to remind them to renew their plan
+    * Send messages to check their progress and how the program is going
+    * Share the workout programs and tips through the app
+
+- Client are able to 
+    * See all the files shared by his trainer
+    * Ask questions / Message with the coach
+    * Upload videos of executions to receive feedback
+    * See his membership/expiry date and renew from there
+    * Schedule 1:1 sessions in the gym
+<br>
+
+## Dependencies 📦📦📦
+The list of dependencies used in the frontend/Backend are :
+
+```javascript
+    "@testing-library/jest-dom"
+    "@testing-library/react"
+    "@testing-library/user-event"
+    "react"
+    "react-dom"
+    "next"
+    "react-redux"
+    "@reduxjs/toolkit"
+    "@mui/material"
+    "@emotion/react"
+    "@emotion/styled"
+    "styled-components"
+    "swr"
+    "typescript"
+   
+```
+The list of dependencies used for code quality and test are :
+
+```javascript
+    "@testing-library/jest-dom"
+    "@testing-library/react"
+    "eslint"
+    "prettier"
+```
+
+<br>
+
+## How to run 🚀🚀🚀
+* First, copy the url of this project
+* Then open your terminal on your local computer and -
+    - navigate to the location you want to create the project in (eg - home or c:/)
+    - enter the command `git clone <url-of-this-project>`. Alternatively, you can provide the name of the folder you want git to clone the project to with `git clone <url-of-this-project> <name-of-folder>`
+    - `cd` into the project folder
+    - Run `npm install` to install the packages for the frontend
```

Comment:
Since the frontend and backend is not distinct, this would more accurately read :
> Run `npm install` to install the project dependencies.
However, it doesn't really matter.